### DISCUSSION TITLE
How could i parse a pdf file properly and send the content to API to return me a html table

### DISCUSSION THREAD

**Discussion by alvincks** [perplexityai/api-discussion#24] [Q&A]

**Comment by goelakshit543** [comment]
You can use external libraries like llama parse and many other to parse your pdfs llama parse works best for pdf's and then after getting the data in desired format you can use that data to be used as context to your model and get the work done as per your wish by setting the system message and sending this parsed data with query as a context window 
so this way you can give it a try ...

**Comment by dfsklar** [comment]
@alvincks Did you ever come up with a system prompt that guaranteed HTML output?
